src/providers/config/AppConfigurationProvider.tsx

```diff
@@ -0,0 +1,68 @@
+import isEmpty from 'lodash/isEmpty';
+import isNil from 'lodash/isNil';
+import React, { createContext, FunctionComponent, useContext, useMemo, useState } from 'react';
+
+export type VerdaccioOptions = {
+  url_prefix: string;
+  uri?: string;
+  base: string;
+  language?: string;
+  version?: string;
+  protocol?: string;
+  host?: string;
+  scope: string;
+  title: string;
+  logo?: string;
+  primaryColor: string;
+  darkMode: boolean;
+};
+
+type ConfigProviderProps = {
+  configOptions: VerdaccioOptions;
+  setConfigOptions: any;
+};
+
+const defaultValues: ConfigProviderProps = {
+  configOptions: {
+    primaryColor: '#4b5e40',
```

Comment:
we use the same color hex in several files. Should we maybe create a const and reuse it? then it's easy to update it in the future